feat: Experimental feature - Duplicate workflow job event to extra queue (#2268)
philips-labs/terraform-aws-github-runner: README.md

  - [Idle runners](#idle-runners)
  - [Ephemeral runners](#ephemeral-runners)
  - [Prebuilt Images](#prebuilt-images)

>   - [Secondary Queue](#secondary-queue)

Reviewer: Still the wrong title here :(

Author: upadted.

### Prebuilt Images

This module also allows you to run agents from a prebuilt AMI to gain faster startup times. You can find more information in [the image README.md](/images/README.md)

### Secondary Queue

The standard queue is an internal queue used by the scale-up lambda, the format we use here is an internal format.
```
export interface GithubWorkflowEvent {
  workflowJobEvent: WorkflowJobEvent;
}
```

> which is extendible format to add more fields to be published along with the workflow job event.

Reviewer: ```suggestion
This extendible format allows to add more fields to be added if needed.
```

> On the second queue we just forward the incoming github event. Which can be used for metrics. You can configure `webhook_events_workflow_job_queue` to enable this experimental feature.

Reviewer: Suggested also to rename the parameters, would require some more changes.

This module also allows you to run agents from a prebuilt AMI to gain faster startup times. You can find more information in [the image README.md](/images/README.md)

> ### Secondary Queue

Reviewer: Would call this GItHub Workflow even copy queue. Secondary is to vague

Reviewer: Can you also add a reference or explain the expected message interface?

Reviewer: Can you make also explicit that the feature is experimental

This module also allows you to run agents from a prebuilt AMI to gain faster startup times. You can find more information in [the image README.md](/images/README.md)

### Secondary Queue

> As per the solution architecture, the web hook events for workflow jobs are sent to a primary queue, from where they are consumed by scale up lambda. In addition to that primary queue, this module allows you to send the webhook events to another secondary queue as well where the webhook event can be processed to get useful insights about the workflow jobs. You can configure `webhook_events_secondary_queue` to enable this feature. 

Author: updated.
